smtp/smtpc.h

```diff
@@ -112,10 +112,22 @@ struct smtpc_request {
 	int body_size;
 	/** Buffer read position. */
 	char *body_rpos;
-	/** SMTP status code. */
+	/** SMTP status code.
+	 * It takes the value of -1 if SMTP code doesn't match to error status.
+	 */
```

Comment:
We use two styles of comments (for one-line and multi-line comments):

```c
/* Hello. */
```

```c
/*
 * Hello,
 * world.
 */
```

(Here the opening `/*` is `/**` for documentation comments.)

If you're curious, Linus about comments style: https://lkml.org/lkml/2016/7/8/625

Reply:
Thank you for explanation of comments style! But I see that other fields of this structrure are described with documentation comments. Is that okay, if those fields will have different types of comments?

Reply:
Fixed.